so101_follower_ros2_bridge:
  ros__parameters:
    port: "/dev/ttyACM1"
    id: "Tzili"
    # calibration_dir: <abs path to your calibration dir>
    use_degrees: true
    max_relative_target: 10 # default is none
    disable_torque_on_disconnect: true
    publish_rate: 30.0

